src/Listener/ApiListener.php

```diff
@@ -151,7 +152,8 @@ protected function _checkRequestMethods()
      */
     public function registerExceptionHandler()
     {
-        Configure::write('Error.exceptionRenderer', 'Crud\Error\ExceptionRenderer');
+        $config = ['exceptionRenderer' => 'Crud\Error\ExceptionRenderer'];
```

Comment:
The problem with this is any other non-default settings are lost. There's an open PR/dicussion related to this cakephp/app#183